integration/zapier/test/authentication.test.js

```
@@ -0,0 +1,37 @@
+    })
+      .get('/api/profile/api-tokens')
+      .reply(200, {
+        '@context': '/api/contexts/Collection',
+        'hydra:member': [{ id: 'tok_1', name: 'Zapier' }],
+      });
+
+    const result = await tester(App.authentication.test, makeBundle());
+    expect(result['hydra:member']).toBeDefined();
+  });
+
+  it('throws RefreshAuthError on a 401', async () => {
```

ESLint

[error] 'it' is not defined.

(no-undef)